projects/igniteui-angular/src/lib/core/styles/components/chip/_chip-theme.scss

```diff
@@ -319,32 +307,102 @@
         }
 
         touch-action: none;
+        outline: none;
 
-        &:focus {
-            outline-style: none;
+        // The focus and hover are build that way since the host is the focusable element
 
+        &:hover {
             %igx-chip__item {
-                @extend %igx-chip__item--focus;
+                color: var-get($theme, 'hover-text-color');
+                background: var-get($theme, 'hover-background');
+                border-color: var-get($theme, 'hover-border-color');
             }
 
             %igx-chip__item--selected {
-                @extend %igx-chip__item--focus-selected;
+                color: var-get($theme, 'hover-selected-text-color');
+                background: var-get($theme, 'hover-selected-background');
+                border-color: var-get($theme, 'hover-selected-border-color');
             }
         }
 
-        &:hover {
+        &:focus {
             %igx-chip__item {
-                @extend %igx-chip__item--hover;
+                outline-style: none;
+                color: var-get($theme, 'focus-text-color');
+                background: var-get($theme, 'focus-background');
+                border-color: var-get($theme, 'focus-border-color');
+                box-shadow: $box-shadow-focus;
             }
 
             %igx-chip__item--selected {
-                @extend %igx-chip__item--hover-selected;
+                color: var-get($theme, 'focus-selected-text-color');
+                background: var-get($theme, 'focus-selected-background');
+                border-color: var-get($theme, 'focus-selected-border-color');
+                transition: $transition;
+                box-shadow: $box-shadow-focus-selected;
             }
         }
 
-        // We target the tag selector directly to
-        // avoid having to set custom classes on the prefix
-        // and suffix based on the host element.
+        @if $variant == 'indigo-design' {
+            &:focus:hover {
+                %igx-chip__item {
+                    color: var-get($theme, 'hover-text-color');
+                    background: var-get($theme, 'hover-background');
+                    border-color: var-get($theme, 'hover-border-color');
+                }
+
+                %igx-chip__item--selected {
+                    color: var-get($theme, 'hover-selected-text-color');
+                    background: var-get($theme, 'hover-selected-background');
+                    border-color: var-get($theme, 'hover-selected-border-color');
+                }
+            }
+        }
+    }
+
+    %igx-chip__item {
+        display: grid;
```

Comment:
I think because of the use of display: grid, currently the grid resizing does not autosize the filter header cells with the default filter chips. 
The main issue is that when the grid autosizes a column(either through api or double click), it resets the header group cell styles like `width` and `minWidth` temporarily and then calculates the size however the browser renders the headers. Here's exact [link](https://github.com/IgniteUI/igniteui-angular/blob/master/projects/igniteui-angular/src/lib/core/utils.ts#L287)

This can be reproduced on the Grid Filtering demo, if you switch to filter row and double click the firs column.
If you know a way this can be dealt with through styles only?

Reply:
Test with the last commit please, the problem should be solved.

Reply:
Thanks, that seem to did it. I've updated the tests and all should be fine now.